pattern-1/bosh-release/jobs/wso2is/templates/repository/conf/tomcat/catalina-server.xml.erb

```diff
@@ -38,12 +38,14 @@
                    connectionUploadTimeout="120000"
                    maxKeepAliveRequests="200"
                    acceptCount="200"
-                   server="WSO2 Carbon Server"
+                   server="<%= p('route_registrar.routes')[0]['uris'][0] %> Server"
                    compression="on"
                    compressionMinSize="2048"
                    noCompressionUserAgents="gozilla, traviata"
                    compressableMimeType="text/html,text/javascript,application/x-javascript,application/javascript,application/xml,text/css,application/xslt+xml,text/xsl,image/gif,image/jpg,image/jpeg"
-                   URIEncoding="UTF-8"/>
+                   URIEncoding="UTF-8"
+                   ciphers="TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256,TLS_DHE_RSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_DHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_DHE_RSA_WITH_AES_128_GCM_SHA256"
```

Comment:
Since this is the HTTP connector, there is no point configuring `ciphers` here. It's only required and used in HTTPS connector.

Reply:
Fixed with https://github.com/wso2/pivotal-cf-is/pull/87/commits/587a37948093541b42b4cd7b8026752ae306950b